cmds/ps/ps_test.go

```diff
@@ -81,7 +90,8 @@ Uid:	110	110	110	110
 			}
 		}
 		c := testutil.Command(t, "aux")
-		c.Env = append(c.Env, "UROOT_PSPATH="+filepath.Join(d, "*/stat"))
+		psp := fmt.Sprintf("UROOT_PSPATH=%s:%s", d, filepath.Join(d, "srv"))
```

Comment:
why an env var and not a flag? flag has the added utility of being listed when you do a -h

Reply:
the env variable is inherited so, e.g., you can set up an environment for a shell and subshells and the behavior will work, even should you (e.g.) run a ps command from vi or emacs or whatever.

in plan 9 we would do this with union mount over /proc, but this is unix and the major way unix people get the effect of union mounts is environment variables with : separated paths: PATH, GOPATH, and so on.